spring-integration-ip/src/test/java/org/springframework/integration/ip/tcp/connection/TcpMessageMapperTests.java

```diff
@@ -458,5 +427,19 @@ public void testWithBytesMapper() {
 		assertThat(message.getHeaders().get(IpHeaders.REMOTE_PORT)).isEqualTo(1234);
 		assertThat(message.getHeaders().get(IpHeaders.CONNECTION_ID)).isEqualTo("someId");
 	}
+	public TcpConnection CreatMockTcpConnection(Object bytes, String hostName, String ipAdress, int port){
```

Comment:
These new methods are not supposed to be `public`. They are concern of the logic just of this class.
So, `private` and `static`.

And our convention is to use camel case for method names.
Not sure from where you took an idea to have it capitalized.
More over it is not consistent since the next method is indeed in camel case...

Please, also run `./gradlew :spring-integration-ip:check` to be sure that you don't break any Checkstyle rules.
In the end, please, add your name to the `@author` list of this class.

Reply:
Thank you very much for your correction. I have finished these modifications. If there are any problems, please feel free to let me know.